yarn.lock

```diff
@@ -2727,7 +2727,7 @@ chalk@^3.0.0:
     ansi-styles "^4.1.0"
     supports-color "^7.1.0"
 
-chalk@^4.0.0, chalk@^4.1.0:
+chalk@^4.0.0, chalk@^4.1.0, chalk@^4.1.1:
```

Comment:
let's revert these changes and run yarn install again. I think you have these because of not using exact version numbers